### PR TITLE
Partial fix for issue/#1223

### PR DESCRIPTION
**Pull request by dancgray**
https://github.com/adaptlearning/adapt_framework/issues/1223

Credit should go to @oliverfoster

### CODE REVIEW

**Comment by oliverfoster** [comment]
+1 untested in ie8

**Comment by oliverfoster** [comment]
Ta Dan

**Comment by hrajotia** [comment]
+1

**Comment by oliverfoster** [comment]
above pr comes from this article > http://javascript.crockford.com/memory/leak.html

**Comment by brian-learningpool** [comment]
+1

**Comment by hrajotia** [comment]
+1

File: js/adapt-contrib-media.js [hunk 1]
```diff
@@ -2,11 +2,30 @@ define(function(require) {
...
     var mep = require('components/adapt-contrib-media/js/mediaelement-and-player');
     require('components/adapt-contrib-media/js/mediaelement-and-player-accessible-captions');
-    
+
     var ComponentView = require('coreViews/componentView');
     var Adapt = require('coreJS/adapt');
 
     var froogaloopAdded = false;
+    
+    function purge(d) {
```
Comment (moloko) [inline js/adapt-contrib-media.js]: this really needs an explanatory comment!

**Comment by oliverfoster** [comment]
+1

**Comment by RobertPeek** [comment]
There is a console error now when leaving the page:

MEDIA12899: AUDIO/VIDEO: Unknown MIME type.

Caused by the addition of this.mediaElement.src = "";

**Comment by oliverfoster** [comment]
warning or error? which browsers?

**Comment by RobertPeek** [comment]
It was an error on IE11

**Comment by oliverfoster** [comment]
i can't replicate at all

**Comment by RobertPeek** [comment]
It appears to only throw the error if the video has been played, and then you leave the page.

**Comment by oliverfoster** [comment]
I tried, unplayed, played, playing, paused and ended but couldn't get it to error, very odd.

**Pull request merged**